active/progress/ProgressRing2.md

```diff
@@ -0,0 +1,146 @@
+filling it when Value gets to Maximum.
+
+```xml
+<ProgressRing IsActive="True" IsIndeterminate="False"
+    Minimum="32" Maximum="212" Value="{x:Bind LiquidTemp}" />
+```
+![](images/ProgressRing-determinate.PNG)
+
+
+## Custom animation
+
+Show progress ring that visually looks like a thermometer boiling over.
+See [AnimatedVisualPlayer](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.AnimatedVisualPlayer)
+for more information on animated visuals.
+
+> Picture would be nice
+
+```xml
+<ProgressRing IsActive="True" >
+    <ProgressRing.IndeterminateSource>
+        <animatedvisuals:LottieProgress />
+    </ProgressRing.IndeterminateSource>
+</ProgressRing>
+```
+
+# API Notes
+
+ProgressRing can now be used in scenarios where progress is determinate or indeterminate. 
+With this additional capability, 
+the determinate ProgressRing does not represent a "hung" state where the end user cannot interact
+with the app. 
+
+Previously, guidance recommended that ProgressRing only be used when the user cannot continue to interact with the app, 
+but this is no longer the only use case and ProgressRing can be used in scenarios where user interaction can continue 
+while the ring is spinning. An example of this would be when a video is loading on a page and a progress ring is displayed 
+while the media is loading. The end user can still interact with other UI elements on the page while they wait 
+for the media to load.
+
+In addition to the updated guidance, the sources for the indeterminate and determinate mode have been exposed so you can 
+choose the animation file that can be played. This will give you the opportunity to customize the look and feel of your
+progress ring and potentially have different animations for determinate vs. indeterminate. 
+
+|Name | Type | Description | Default | 
+|:--|:-:|:-:|:-:|
+| IsIndeterminate | boolean | Gets or sets a value that indicates whether the progress ring reports generic progress with a repeating pattern (indeterminate progress) or reports progress based on the Value property (determinate progress) | True | 
+| IsActive | boolean | Gets or sets a value that indicates whether the progress ring is being displayed to the end user. If IsActive is false, the ProgressRing is not shown, but space is reserved for it in the UI layout. To not reserve space for it, set its Visibility property to Collapsed. | True |
+| Value | double | Gets or sets the current setting of the range control, which may be coerced. The value ranges from 0 to 100??. Has no effect when `IsIndeterminte` is true. | 0-100 | 
+| Maximum | double | Gets or sets the highest possible `Value` of the range element. Has no effect when `IsIndeterminate` is true.
+| Minimum | double | Gets or sets the lowest possible `Value` of the range element. Has no effect when `IsIndeterminate` is true.
+| DeterminateSource | IAnimatedVisualSource | Gets or sets the animation file used for the determinate state of the ring. | N/A | 
```

Comment:
What does N/A mean? If I read the DeterminateSource property on a freshly-constructed ProgressRing, what do I get? I'm guessing `null`.